utils/singlejs/README.md

```diff
@@ -0,0 +1,38 @@
+## Bundling Chime SDK into a single .js file
+
+This demo shows how to bundle the latest release of `amazon-chime-sdk-js` into a single JS file using Rollup. You'll need to have Node 10+ and npm 6.11+ on your local development machine.
```

Comment:
Node 10 is no longer supported. Can we update to Node 14 or 16?

Reply:
Updated it to `14+`. Thanks for catching!